adapters/Unity/UnityAdapter/GADMUnityInterstitialAd.m

```diff
@@ -118,73 +118,28 @@ - (void)presentInterstitialFromRootViewController:(UIViewController *)rootViewCo
     return;
   }
 
-  if (![UnityAds isReady:_placementID]) {
-    NSError *error = GADMAdapterUnityErrorWithCodeAndDescription(
-        GADMAdapterUnityErrorShowAdNotReady, @"Failed to show Unity Ads interstitial.");
-    [strongConnector adapter:strongAdapter didFailAd:error];
-    return;
+  if (!_placementID) {
```

Comment:
This is not invoking any failure callback. I'm assuming that [UnityAds show:rootViewController placementId:nil] is expected to fail and UnityAds SDK will invoke the unityAdsShowFailed callback?

Reply:
This is correct.  If the placement ID is nil, `unityAdsShowFailed` will be called with the error  kUnityShowErrorInvalidArgument.

Allowing show to be called here will help provide more insight into issues related to placement fill.  If a placement had been loaded but is no longer ready `unityAdsShowFailed` will be called with the error `kUnityShowErrorNotReady`.

Reply:
acknowledged. passing to UnityAds SDK also lets you get insight into how often this might be happening.